README.rst

```diff
@@ -20,7 +20,7 @@ Using Docker
 ------------
 
 * Install Docker
-* ``pip install docker-compose``
+* ``pip install docker compose``
```

Comment:
This line can actually be removed, since `compose` is now a part of `docker` itself